src/zappy_server_src/core/commands/incantation_command.c

```diff
@@ -140,6 +139,7 @@ server_data_t *serv)
         print_retcode(222, res, player->player_peer, true);
     else
         print_retcode(316, NULL, player->player_peer, false);
+    entity_diff_add_entity(serv->modified_entities, player_entity);
```

Comment:
Move it to the incantation action function or inside the IF (Otherwise, it will send the entity even the verification have failed.)